From 9c1aa796d8e32346293466ba1a62474375a994a9 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Tue, 29 Sep 2020 19:33:29 +0800
Subject: [PATCH] ARM: dts: rockchip: rk628: add ref clk for gvi

GVI only can use VOP sclk as combtxphy reference clock.

Change-Id: I919bdceaac6d7bb97658d958fecb52971117c82b
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 arch/arm/boot/dts/rk628.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rk628.dtsi b/arch/arm/boot/dts/rk628.dtsi
index 8c724f9c5d57..b2a2f9410cc0 100644
--- a/arch/arm/boot/dts/rk628.dtsi
+++ b/arch/arm/boot/dts/rk628.dtsi
@@ -240,8 +240,8 @@
 
 	rk628_combtxphy: combtxphy {
 		compatible = "rockchip,rk628-combtxphy";
-		clocks = <&rk628_cru CGU_PCLK_TXPHY_CON>;
-		clock-names = "pclk";
+		clocks = <&rk628_cru CGU_PCLK_TXPHY_CON>, <&rk628_cru CGU_SCLK_VOP>;
+		clock-names = "pclk", "ref_clk";
 		resets = <&rk628_cru RGU_TXPHY_CON>;
 		#phy-cells = <0>;
 		status = "disabled";
-- 
2.35.3

